html/components/team-time-tab.js

```diff
@@ -981,7 +989,11 @@ function createJamDialog() {
 		currentPeriod = v;
 		dialog.find("table.Period.Show").removeClass("Show");
 		dialog.find("table.Period[nr="+v+"]").addClass("Show");
-	})
+	});
+	
+	WS.Register(['ScoreBoard.Jam(*).Number'], function(k, v) {
+		nextJam = v;
```

Comment:
Hmm, is this safe? It's going to get deleted jams wrong at least.

Reply:
`ScoreBoard.Jam()` is the upcoming Jam, so this will not be triggered by any changes to past jams (unless they change the number of the upcoming Jam).

But a guard for deletion makes sense - added (and removed the debug code).